src/zope/i18n/locales/xmlfactory.py

```diff
@@ -1251,6 +1275,15 @@ def _extractDelimiters(self):
           ...     <alternateQuotationStart>`</alternateQuotationStart>
           ...     <alternateQuotationEnd>'</alternateQuotationEnd>
           ...   </delimiters>
+          ...   <identity>
+          ...     <version number="1.0"/>
+          ...     <generation date="2003-12-19" />
+          ...     <language type="en" />
+          ...     <territory type="US" />
+          ...     <variant type="POSIX" />
+          ...   </identity>
+          ... </ldml>'''
+
           ... </ldml>'''
```

Comment:
This looks wrong.

Reply:
Hah! Yeah. copy-and-paste error. Doctest doesn't mind, though. (Which is sad, now that I think about it.)